src/wp-includes/block-template-utils.php

```diff
@@ -432,6 +432,41 @@ function _inject_theme_attribute_in_block_template_content( $template_content )
 	return $template_content;
 }
 
+/**
+ * Parses wp_template content and removes the theme attribute from
+ * each wp_template_part
+ *
+ * @access private
+ * @since 5.9.0
+ *
+ * @param string $template_content serialized wp_template content.
+ *
+ * @return string Updated wp_template content.
+ */
```

Comment:
```suggestion
/**
 * Parses a block template and removes the theme attribute from each template part.
 *
 * @access private
 * @since 5.9.0
 *
 * @param string $template_content Serialized block template content.
 * @return string Updated block template content.
 */
```

Switches to plain language descriptions rather than the technical name of the object. 

Core doesn't include a space between the `@param` and `@return` lines in docblocks.